src/credentials/loginManager.ts

```diff
@@ -43,8 +44,8 @@ export class LoginManager {
                 throw new Error(`No credentials found for id ${asString(credentialsProviderId)}`)
             }
 
-            // TODO : Get a region relevant to the partition for these credentials -- https://github.com/aws/aws-toolkit-vscode/issues/188
-            const accountId = await getAccountId(storedCredentials.credentials, 'us-east-1')
+            const credentialsRegion = provider.getDefaultRegion() ?? this.defaultCredentialsRegion
```

Comment:
Assume this requires that the credentials in China and other partitions must specify their region. Is this common?

Reply:
This is my understanding. For example, if you're using the AWS CLI, you're either explicitly setting the region command line arg for each call, or you're setting region in your credentials.